pkg/cluster/connectionsManager.go

```diff
@@ -45,12 +44,12 @@ func IsKCInstalledOnCluster(ctx context.Context, c *KubernetesCluster) (bool, er
 	return true, data.Error()
 }
 
-// CheckIfConnectionsExist checks if the Kafka connections exist
-func CheckIfConnectionsExist(ctx context.Context, c *KubernetesCluster, namespace string, k *kafkamgmtclient.KafkaRequest) error {
+// CheckIfKafkaConnectionExists checks if the Kafka connections exist
+func CheckIfKafkaConnectionExists(ctx context.Context, c *KubernetesCluster, namespace string, k string) error {
```

Comment:
We probably need generic function that accepts parameters based of service type

Reply:
You can pass an interface and check its type using a type switch to achieve this:

https://tour.golang.org/methods/16

Reply:
In this case interface might not be that important to have but this comment is gold for switch case we have:
https://github.com/redhat-developer/app-services-cli/pull/975#discussion_r703249531